docs/UserGuide.md

```diff
@@ -170,7 +167,7 @@ Examples:
 
 Edits an existing volunteer in the volunteer list.
 
-Format: `vedit VOLUNTEER_ID [n/NAME] [p/PHONE] [e/EMAIL] [s/SKILL]…​`
+Format: `vedit VOLUNTEER_INDEX [n/NAME] [p/PHONE] [e/EMAIL] [s/SKILL]…​`
```

Comment:
Wait do we use index or id?

Reply:
we're using index. this is a clarity suggestion from #174